remix.init/_app_redirects

```diff
@@ -0,0 +1,10 @@
+# This file is being used instead of the typical Netlify _redirects file in the root of your application
```

Comment:
I renamed it from `_custom_redirects` to `_app_redirects` @whitep4nth3r. I think it conveys what the file is better along with the comments in the file. Thoughts on the wording? cc: @stephmarie17

Reply:
Much better I think!